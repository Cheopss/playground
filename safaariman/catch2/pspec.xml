<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>catch2</Name>
        <Homepage>https://github.com/catchorg/catch2</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Packager>
        <License>BSLv1</License>
        <IsA>library</IsA>
        <Summary>C++ Automated Test Cases in Headers</Summary>
        <Description>Catch2 is a multi-paradigm test framework for C++, which also supports Objective-C. It is primarily distributed as a single header file, although certain extensions may require additional headers.</Description>
        <Archive sha1sum="dbf1bee2391a58936e353f67c6935ecdfbae336b" type="targz">https://github.com/catchorg/Catch2/archive/v2.11.1.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>cmake</Dependency>
            <Dependency>python3</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>catch2</Name>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="library">/usr/lib</Path>
            <Path fileType="doc">/usr/share/doc</Path>
            <Path fileType="data">/usr/share</Path>
        </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2020-03-17</Date>
            <Version>2.11.1</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
    </History>
</PISI>
