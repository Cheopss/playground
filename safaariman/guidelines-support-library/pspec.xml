<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>guidelines-support-library</Name>
        <Homepage>https://github.com/Microsoft/GSL</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Packager>
        <License>MIT</License>
        <IsA>library</IsA>
        <Summary>Microsoft Guidelines Support Library</Summary>
        <Description>The Guideline Support Library (GSL) contains functions and types that are suggested for use by the C++ Core Guidelines maintained by the Standard C++ Foundation. This package contains Microsoft's implementation of GSL.</Description>
        <Archive sha1sum="37833349b0f0a336eac3b680e79ffd78b50ff5c9" type="targz">https://github.com/microsoft/GSL/archive/v2.1.0.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>cmake</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>guidelines-support-library-devel</Name>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="doc">/usr/share/doc</Path>
        </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2020-03-18</Date>
            <Version>2.1.0</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
    </History>
</PISI>
