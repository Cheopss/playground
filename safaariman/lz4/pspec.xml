<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>lz4</Name>
        <Homepage>https://lz4.github.io/lz4/</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Packager>
        <License>BSD</License>
        <IsA>library</IsA>
        <Summary>Extremely fast compression algorithm</Summary>
        <Description>LZ4 is an extremely fast loss-less compression algorithm, providing compression speed at 400 MB/s per core, scalable with multi-core CPU. It also features an extremely fast decoder, with speed in multiple GB/s per core, typically reaching RAM speed limits on multi-core systems.</Description>
        <Archive sha1sum="4dc36f29d7b4e5b45b114be2674068200714abb3" type="targz">https://github.com/lz4/lz4/archive/v1.9.2.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>make</Dependency>
            <Dependency>glibc-devel</Dependency>
            <Dependency>diffutils</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>lz4</Name>
        <RuntimeDependencies>
            <Dependency>glibc</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="executable">/usr/bin</Path>
            <Path fileType="library">/usr/lib/liblz4*</Path>
            <Path fileType="man">/usr/share/man</Path>
        </Files>
    </Package>

    <Package>
        <Name>lz4-devel</Name>
        <Summary>Development files for lz4</Summary>
        <RuntimeDependencies>
            <Dependency release="current">lz4</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="data">/usr/lib/pkgconfig</Path>
        </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2020-03-19</Date>
            <Version>1.9.2</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
    </History>
</PISI>
