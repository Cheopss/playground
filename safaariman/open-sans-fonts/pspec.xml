<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>open-sans-fonts</Name>
        <Homepage>https://www.opensans.com/</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Packager>
        <License>Apache-2.0</License>
        <IsA>data:font</IsA>
        <Summary>Open Sans is a humanist sans-serif typeface designed by Steve Matteson</Summary>
        <Description>Open Sans is a humanist sans serif typeface designed by Steve Matteson, Type Director of Ascender Corp. This version contains the complete 897 character set, which includes the standard ISO Latin 1, Latin CE, Greek and Cyrillic character sets. Open Sans was designed with an upright stress, open forms and a neutral, yet friendly appearance. It was optimized for print, web, and mobile interfaces, and has excellent legibility characteristics in its letter forms.</Description>
        <Archive sha1sum="3edba27a9951a4cfd18af1c1f9d3df9a43f08ace" type="zip">https://www.opensans.com/download/open-sans.zip</Archive>
        <Archive sha1sum="baac0e7cb19ab8eaa6b41d4ac8258d6731ed38d7" type="zip">https://www.opensans.com/download/open-sans-condensed.zip</Archive>
    </Source>

    <Package>
        <Name>open-sans-fonts</Name>
        <Files>
            <Path fileType="doc">/usr/share/doc</Path>
            <Path fileType="data">/usr/share/fonts</Path>
        </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2020-03-20</Date>
            <Version>1.10</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
    </History>
</PISI>
