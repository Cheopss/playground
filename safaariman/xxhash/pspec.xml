<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>xxhash</Name>
        <Homepage>https://cyan4973.github.io/xxHash/</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Packager>
        <License>BSD</License>
        <License>GPLv2</License>
        <IsA>library</IsA>
        <Summary>Extremely fast hash algorithm</Summary>
        <Description>xxHash is an Extremely fast Hash algorithm, running at RAM speed limits. It successfully completes the SMHasher test suite which evaluates collision, dispersion and randomness qualities of hash functions. Code is highly portable, and hashes are identical on all platforms (little / big endian).</Description>
        <Archive sha1sum="f05904b4877fdedfae2a62391245a41e433cbc44" type="targz">https://github.com/Cyan4973/xxHash/archive/v0.7.3.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>make</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>xxhash</Name>
        <Files>
            <Path fileType="executable">/usr/bin</Path>
            <Path fileType="library">/usr/lib</Path>
            <Path fileType="man">/usr/share/man</Path>
            <Path fileType="doc">/usr/share/doc</Path>
        </Files>
    </Package>

    <Package>
        <Name>xxhash-devel</Name>
        <Summary>Development files for xxhash</Summary>
        <RuntimeDependencies>
            <Dependency release="current">xxhash</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="header">/usr/include</Path>
        </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2020-03-19</Date>
            <Version>0.7.3</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
    </History>
</PISI>
