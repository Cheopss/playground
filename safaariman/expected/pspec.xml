<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>expected</Name>
        <Homepage>https://github.com/TartanLlama/expected</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Packager>
        <License>CC0</License>
        <IsA>library</IsA>
        <Summary>Single header implementation of std::expected with functional-style extensions.</Summary>
        <Description>std::expected is proposed as the preferred way to represent object which will either have an expected value, or an unexpected value giving information about why something failed. Unfortunately, chaining together many computations which may fail can be verbose, as error-checking code will be mixed in with the actual programming logic. This implementation provides a number of utilities to make coding with expected cleaner.</Description>
        <Archive sha1sum="2ce5120ef6b4238b092a73214a48516462f809ce" type="targz">https://github.com/TartanLlama/expected/archive/v1.0.0.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>cmake</Dependency>
            <Dependency>git</Dependency>
            <Dependency>catch2</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>expected-devel</Name>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="doc">/usr/share/doc</Path>
            <Path fileType="data">/usr/share/cmake</Path>
        </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2020-03-17</Date>
            <Version>1.0.0</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
    </History>
</PISI>
